code/go/0chain.net/smartcontract/zcnsc/nodes_reader.go

```diff
@@ -51,6 +51,19 @@ func GetGlobalSavedNode(ctx state.CommonStateContextI) (*GlobalNode, error) {
 	}
 }
 
+func InitConfig(ctx state.StateContextI) error {
+	node := &GlobalNode{ID: ADDRESS}
+	err := ctx.GetTrieNode(node.GetKey(), node)
+	switch err {
+	case nil, util.ErrValueNotPresent:
+		node.ZCNSConfig = getConfig()
```

Comment:
Do not insert if err is nil, which means the config exist, the insert will overwritten the config.